apps/client/src/services/load_results.ts

```diff
@@ -44,9 +44,17 @@ interface OptionRow {}
 
 interface NoteReorderingRow {}
 
-interface ContentNoteIdToComponentIdRow {
+interface NoteEmbeddingRow {
```

Comment:
Don't be scared of this change - the `diff` shows this way because the interface was duplicated twice